scripts/es-integration-test.sh

```diff
@@ -68,7 +69,7 @@ wait_for_it() {
     ''%{http_code}''
   )
   local counter=0
-  while [[ "$(curl ${params[@]} ${url})" != "200" && ${counter} -le 30 ]]; do
+  while [[ "$(curl ${params[@]} ${url})" != "200" && ${counter} -le 60 ]]; do
```

Comment:
1min (2*30) seems insufficient sometimes, we can wait longer if it takes OS longer to start